src/textual/widgets/_tree.py

```diff
@@ -692,14 +692,18 @@ def get_label_width(self, node: TreeNode[TreeDataType]) -> int:
         label = self.render_label(node, NULL_STYLE, NULL_STYLE)
         return label.cell_len
 
+    def clear_line_cache(self) -> None:
```

Comment:
This should be private, i.e. prefixed with `_`. It's not something that the dev needs to know about generally.